src/utils/handle-request.ts

```diff
@@ -1,29 +1,19 @@
-export default async function handleRequest(request: Request, edgeCacheTtl: number) {
+export default async function handleRequest(request: Request, edgeCacheTtl: number): Promise<Response> {
+  // https://developers.cloudflare.com/workers/runtime-apis/request/#requestinitcfproperties
   const cfOptions = edgeCacheTtl > 0 ? {
-    // Requests proxied through Cloudflare are cached even without Workers according to a zone’s default
-    // or configured behavior. Setting Cloudflare cache rules to further customised the behavior
     cf: {
-      // NOTE: cf is Enterprise only feature
```

Comment:
@samkahchiin I did not find in the documentation that this would be an enterprise-only feature, maybe it is publicly available now?